explore/src/main/scala/explore/Explore.scala

```diff
@@ -201,13 +200,14 @@ object ExploreMain {
               (StateProviderComponent((rootComponent _).reuseAlways): VdomNode).reuseAlways
             ): VdomNode).reuseAlways
           ).renderIntoDOM(container)
-
-          releaseDispatcher
         }
       }
-      .onError { t =>
+      .void
+      .handleErrorWith { t =>
         logger.error(t)("Error initializing") >>
           crash(s"There was an error initializing Explore:<br/>${t.getMessage}")
       }
   }
+
+  // TODO Release dispatcher on stop.
```

Comment:
Given the nature of web apps, I think we can get away with not implementing this.

Reply:
Right, I wasn't sure about that